test/cypress/integration/tests/scenario-exec-spec.js

```diff
@@ -48,44 +48,117 @@ import {
 } from '../../../../js-apps/meep-frontend/src/js/util/elem-utils';
 
 // Import Test utility functions
-import { selector, click, type, select, verify, verifyEnabled, verifyForm } from '../util/util';
+import { selector, click, check, type, select, verify, verifyEnabled, verifyForm, verifyChecked } from '../util/util';
 
 // Scenario Execution Tests
 describe('Scenario Execution', function () {
 
   // Test Variables
   let defaultScenario = 'None';
   let sandbox = 'sbox-test';
-
-  // Test Setup
-  beforeEach(() => {
```

Comment:
the beforeEach doesn;t seem to always work. So some test cases sometimes fail and we need to run cypress again because it tried to execute the test case but somehow didn't run the portion in the before each. The content of the beforeEach was moved to a function that is invoked every time now... and cypress always work as expected (otherwise, trying to click some buttons not already enabled)...